Build/Build.cs

```diff
@@ -287,7 +287,12 @@ class Build : NukeBuild
         });
 
     bool HasDocumentationChanges =>
-        Changes.Any(x => x.StartsWith("docs"));
+        Changes.Any(x => 
+            x.StartsWith("docs") || 
+            x.StartsWith("CONTRIBUTING.md") ||
+            x.StartsWith("cSpell.json") ||
+            x.StartsWith("LICENSE") ||
+            x.StartsWith("README.md"));
 
     bool HasSourceChanges =>
```

Comment:
I think this needs to be structured something like this to ensure that `HasSourceChanges` is the complement of `HasDocumentationChanges`.

```cs
bool HasDocumentationChanges => Changes.Any(x => x.IsDocumentation());

bool HasSourceChanges=> Changes.Any(x => !x.IsDocumentation());

static bool IsDocumentation(string x)
{
    return x.StartsWith("docs") || 
           x.StartsWith("CONTRIBUTING.md") ||
           x.StartsWith("cSpell.json") ||
           x.StartsWith("LICENSE") ||
           x.StartsWith("README.md");
}
```

Reply:
Yes, I wanted to refactor this, but focused on the "build clean up PR".

Now I can do that